changelog/unreleased/kong/bump-ngx-wasm-module.yml

```
@@ -0,0 +1,2 @@
+message: "Bump `ngx_wasm_module` to `ae7b61150de7c14eb901307daed403b67f29e962`"
```

```suggestion
message: "Bumped `ngx_wasm_module` to `ae7b61150de7c14eb901307daed403b67f29e962`"
```
This was changed all over the changelog during the last release.